src/app/reports/menu/menu.html

```diff
@@ -11,12 +11,7 @@
       <span>Utility</span>
     </h4>
   </a>
-  <a ui-sref="reports.layout.facility" ui-sref-active="active" class="list-group-item">
```

Comment:
it was a duplicate, i think, i confirmed from my other team members, Musa before removing it